src/executables/util_service.c

```diff
@@ -1931,19 +1934,19 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
 }
 
 /*
- * process_server_start_single_node - start both a page and a transaction server,
- *                                    the page server must be started before transaction server
+ * process_servers_within_node -  Start both a page and a transaction server,
+ *                                the page server must be started before transaction server
  */
 static int
-process_server_start_single_node (char *server_name, int command_type)
+process_servers_within_node (char *server_name, int command_type, bool ha_mode)
 {
   int status = NO_ERROR;
   int pid = 0;
 
   const char *args_page[] = { UTIL_CUBRID_NAME, "-t", "page", server_name, NULL };
   status = proc_execute (UTIL_CUBRID_NAME, args_page, false, false, false, &pid);
 
-  if (status == NO_ERROR && !is_server_running (CHECK_PAGE_SERVER, server_name, pid))
+  if (status == NO_ERROR && !is_server_running (ha_mode ? CHECK_HA_PAGE_SERVER : CHECK_PAGE_SERVER, server_name, pid))
```

Comment:
`is_server_running()` function checks if there is a pattern matching `CHECK_PAGE_SERVER ("Page-Server")` in the process status information provided by cub_master, in order to determine whether the server is running or not.
And the status information is used to print out to the user.

Reusing the `CHECK_PAGE_SERVER` pattern means that cub_master should output the status information without distinguishing between HA and non-HA servers. Similar to how `oldHA` previously differentiated between the `server tdb (rel, pid)` and the `HA-server tdb (rel, pid)` in the output, the I distinguish between `CHECK_PAGE_SERVER` and `CHECK_HA_PAGE_SERVER` to provide the same functionality.

Reply:
TO SUMMARY! 

It seems appropriate to proceed with @hornetmj  suggestion. Initially, I removed the distinction by removing `CHECK_HA_PAGE_SERVER` since I thought that non-HA servers and HA servers cannot be running simultaneously. However, I discovered *exceptional situations where both servers can be running simultaneously. It is possible that these situations were already considered in the oldHA, or it might be a new scenario. Instead of analyzing these exceptions, it might be better to focus on implementing the required functionality.

*Exceptional situations:

1. Session 1 (Terminal 1) and Session 2 (Terminal 2) have different `$CUBRID` 
    - by different commit number or something
2. In each session, both HA server and non-HA server are executed.
3. When checking the status of the Cubrid server, it is observed that both servers are running.

@hornetmj  @wrlawodms  what do you think?

Reply:
@joohok I sorry to make you confution. The guideline is not for collecting other opinions. I just want for you to save the time and focus more important and difficult tasks like supporting fail-over and split-brain mechanism perfectly. Additionally, the sooner the newha function is supported, the sooner QA testing and user-level test verification can be introduced, and the more effective it can be to derive and find overall problems. Therefore, I ask that you minimize unnecessary analysis and proceed with a focus on key features. However, please leave a clear comment on the part that needs to be reviewed later.

Reply:
OK, then I will 
1) rollback these changes, 
2) leave a comment about what to do, (remove distinction between non-HA server and HA-server)
3) and make an issue and link to this issue to track.

Reply:
Nit:

About 3. it depends on you. I think it is enough to leave comments with a keywords like 'JOHOK TODO'. Because you can open a issue grouping several comments later.

Reply:
OK, I agree.

Reply:
I understand that old-HA distinguishes the non-HA server and HA-server, but what is it for? Is it valid for NewHA?

Reply:
@wrlawodms 

> I understand that old-HA distinguishes the non-HA server and HA-server, but what is it for?

The current implementation distinguishes between nonHA servers and HA servers for only one reason.

It is only used for users to determine whether the currently running server is an HA server or a nonHA server based on the output of the `cub_commdb -O` command. There doesn't seem to be any other purpose or utility for it.

I think it would be valid for newHA as well, 
In terms of 
1) providing the same user experience and 
2) being able to catch bugs like the one below analysis 

Analysis :
HA servers and nonHA servers with the same name cannot be started simultaneously.
An example of a problem scenario when not distinguishing between HA servers and non-HA servers is as follows.
(used exceptional case mentioned in https://github.com/CUBRID/cubrid/pull/4400/files#discussion_r1224092830 )

Terminal 1: Create a nonHA server named testdb.
Terminal 2: Create an HA server named testdb.
1. In terminal 1, Start testdb as nonHA.
2. In ternminal 2, Execute "cubrid hb start" and the node's state becomes Master.
   - Without distinguishing between HA and nonHA, it is assumed to provide HA services because HB utility assumes that the HA server is already running..
3. Connect to the server in Terminal 2 and perform transactions. The transaction results will be replicated in the HA-cluster.
   - Since they use the same cub_master, it is possible to connect to the server started in Terminal 1.

Issues:

- If the server in Terminal 2 dies, cub_master will not manage it.
  - If an HA server dies, cub_master should revive it.
- Additionally, the cluster will not recognize this server.

By distinguishing between HA servers and nonHA servers, such bugs can be addressed. It can also be applied to newHA.

For example, when executing "cubrid hb start," it can check if a server with the same name already exists and, if so, kill the existing server and start the HA server instead.

Reply:
@joohok Disccussed off-line, I think customers have no experience the scenario you mentioned(but, need to check). So, there is other chance like block starting non-ha server in ha_mode=on. Let's consider it later.

Reply:
@joohok Thank you for the elaboration. 
> providing the same user experience and

Agreed.

> being able to catch bugs like the one below analysis

I don't think we need to address that kind of potential wrong behavior which has been since the develop branch.